packages/app-mobile/components/ItemList.js

```diff
@@ -81,7 +81,7 @@ class ItemList extends React.Component {
 			itemComps = [blankItem('top', this.state.topItemIndex * this.props.itemHeight)];
 
 			for (let i = this.state.topItemIndex; i <= this.state.bottomItemIndex; i++) {
-				const itemComp = this.props.itemRenderer(items[i]);
+				const itemComp = this.props.itemRenderer(items[i], i);
```

Comment:
As above, this allows us to provide an index to a callback in `ScreenHeader.tsx`.